robolectric/src/main/java/org/robolectric/internal/SdkConfig.java

```diff
@@ -1,40 +1,27 @@
 package org.robolectric.internal;
 
 import android.os.Build;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
+
+import java.util.*;
 import javax.annotation.Nonnull;
 import org.robolectric.internal.dependency.DependencyJar;
 import org.robolectric.util.Logger;
 
-public class SdkConfig implements Comparable<SdkConfig> {
-
-  private static final Map<Integer, SdkVersion> SUPPORTED_APIS = Collections.unmodifiableMap(new HashMap<Integer, SdkVersion>() {
-    private final double jdkVersion = Double.parseDouble(System.getProperty("java.specification.version"));
+public final class SdkConfig implements Comparable<SdkConfig> {
```

Comment:
Should we rename it SdkVersion or ApiVersion or AndroidRuntimeVersion or similar? Config isn't really the right name anymore.

Reply:
Unfortunately it's a public API (despite the package name) so it'd have to be subclassed/deprecated.